x-pack/plugins/apm/public/components/app/service_groups/service_groups_tour.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiButtonEmpty, EuiSpacer, EuiText, EuiTourStep } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ElasticDocsLink } from '../../shared/links/elastic_docs_link';
+
+export type TourType = 'createGroup' | 'editGroup' | 'serviceGroupCard';
+
+interface Props {
+  title: string;
+  content: string;
+  tourEnabled: boolean;
+  dismissTour: () => void;
+  children: React.ReactElement;
+}
+
+export function ServiceGroupsTour({
+  tourEnabled,
+  dismissTour,
+  title,
+  content,
+  children,
+}: Props) {
+  return (
+    <EuiTourStep
+      content={
+        <>
+          <EuiText size="s" color="subdued">
+            {content}
+          </EuiText>
+          <EuiSpacer />
+          <ElasticDocsLink
+            section="/kibana"
+            path="/services.html"
+            target="blank"
+          >
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.apm.serviceGroups.tour.content.link', {
+                defaultMessage: 'Learn more in our documentation',
```

Comment:
```suggestion
                defaultMessage: 'Learn more in the docs.',
```
@bmorelli25 really small detail, but can you confirm that "in the docs" is the right phrase we typically use in Kibana here?

Reply:
We have a few instances of "our docs/documentation" but "the docs/documentation" is much more frequently used. It looks like we use "the docs" just as often as we use "the documentation". We should standardize this 😄.

Reply:
`the docs` is the standard for this feature then 😆

<img width="373" alt="Screen Shot 2022-03-22 at 2 46 10 PM" src="https://user-images.githubusercontent.com/55978943/159553695-da31ff47-e1fb-4943-b0e8-25e37affed3e.png">